examples/browser/package.json

```diff
@@ -21,6 +21,7 @@
     "@theia/metrics": "^0.3.11",
     "@theia/mini-browser": "^0.3.11",
     "@theia/monaco": "^0.3.11",
+    "@theia/monaco-textmate": "^0.3.11",
```

Comment:
I think the textmate stuff could go into the monaco extension.

Reply:
It could, but I felt like this textmate coloring was a feature on its own, that people could work without (and maybe just use Monarch Themes instead with the correct extension). What would be the main reason to merge it in monaco ?